pkg/poseidonhttp/poseidonhttp.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package poseidonhttp
+
+import (
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/kubernetes-sigs/poseidon/pkg/firmament"
+	"github.com/kubernetes-sigs/poseidon/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	pathMetrics = "/metrics"
+	PathHealth  = "/health"
+)
+
+func ServeMetrics(port string, fc firmament.FirmamentSchedulerClient) {
+	metrics.Register()
+	addr := ":" + port
+	metricsMux := http.NewServeMux()
+	HandleMetricsHealth(metricsMux, nil)
+	server := &http.Server{
+		Addr:    addr,
+		Handler: metricsMux,
+	}
+	go glog.Fatal(server.ListenAndServe())
+}
+
+// HandleMetricsHealth registers metrics and health handlers.
+func HandleMetricsHealth(mux *http.ServeMux, fc firmament.FirmamentSchedulerClient) {
+	mux.Handle(pathMetrics, promhttp.Handler())
+	// TODO(jiaxuanzhou): add healthz handler
```

Comment:
What's the problem that we can't support healthz endpoint now?

Reply:
i will implement it in another pr